phpstan-baseline.neon

```diff
@@ -0,0 +1,37 @@
+parameters:
+	ignoreErrors:
+		-
+			message: '#^Creating callable from array\{mixed, non\-falsy\-string\} but it might not be a callable\.$#'
+			identifier: callable.nonCallable
+			count: 2
+			path: src/AuthorizationService.php
+
+		-
+			message: '#^Method Authorization\\Controller\\Component\\AuthorizationComponent\:\:can\(\) should return bool but returns Authorization\\Policy\\ResultInterface\|bool\.$#'
+			identifier: return.type
+			count: 1
+			path: src/Controller/Component/AuthorizationComponent.php
+
+		-
+			message: '#^Method Authorization\\Controller\\Component\\AuthorizationComponent\:\:canResult\(\) should return Authorization\\Policy\\ResultInterface but returns Authorization\\Policy\\ResultInterface\|bool\.$#'
+			identifier: return.type
+			count: 1
+			path: src/Controller/Component/AuthorizationComponent.php
+
+		-
+			message: '#^Method Authorization\\Controller\\Component\\AuthorizationComponent\:\:implementedEvents\(\) should return array\<string, mixed\> but returns array\<int\|string, string\>\.$#'
+			identifier: return.type
+			count: 1
+			path: src/Controller/Component/AuthorizationComponent.php
+
+		-
+			message: '#^Cannot call method getIdentifier\(\) on array\|Authentication\\IdentityInterface\.$#'
```

Comment:
This error should actually be addressed at some point, good for now.

Reply:
we have that prop as `ArrayAccess|array` with native PHP attributes but overwritten with `@var \Authentication\IdentityInterface` because of the nature of it being a decorator implementation.

Haven't taken too much time thinking about it but feels pretty hard to get stan to understand that correctly.

Reply:
Yes it's a tricky one, hence I too have been ignoring it :stuck_out_tongue:.